docs/serving/deploying/private-registry.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Deploying images from a private container registry"
+linkTitle: "Deploying from private registries"
+weight: 10
+type: "docs"
+---
+
+Learn how to configure your Knative cluster to deploy images from a private 
+container registry.
+
+To enable access to your private container images, you create a `imagePullSecret` type 
```

Comment:
suggestion: "create a Kubernetes secret of type `imagePullSecret` with your credentials"

Or, if you leave it the way it is, "create a" --> "create an"